iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/HiveIcebergStorageHandler.java

```diff
@@ -410,64 +413,69 @@ public boolean canSetColStatistics(org.apache.hadoop.hive.ql.metadata.Table hmsT
   }
 
   @Override
-  public boolean setColStatistics(org.apache.hadoop.hive.ql.metadata.Table hmsTable,
-      List<ColumnStatistics> colStats) {
+  public boolean setColStatistics(org.apache.hadoop.hive.ql.metadata.Table hmsTable, List<ColumnStatistics> colStats,
```

Comment:
Since we are changing the API, can we think of returning a map<Column, Boolean Result> OR a generic object containing mapping of (Column Name -> Merge result) (since we cant predict things that this generic object will contain).
This return type is useful in the case of Alter table Change column's datatype.

@simhadri-g @deniskuzZ WDYT?